### PR TITLE
Avoid returning negative seconds value when start date is older than end

### PR DESCRIPTION
**Pull request by lstocchi**
this is linked to https://github.com/redhat-developer/intellij-tekton/issues/205

### CODE REVIEW

File: src/test/java/com/redhat/devtools/intellij/common/utils/DateHelperTest.java [hunk 1]
```diff
@@ -81,4 +81,11 @@ public void shouldOnlyPrintDaysIfDifferenceIsXXXDays() {
         String res = DateHelper.humanizeDate(startTime, endTime);
         assertEquals(res, "374 d ");
     }
+
+    @Test
+    public void shouldOnlyPrint0SecondsIfStartIsOlderThanEnd() {
```
Comment (jeffmaury) [inline src/test/java/com/redhat/devtools/intellij/common/utils/DateHelperTest.java:86]: test is badly name and I think we are missing the test for the negative test

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: src/test/java/com/redhat/devtools/intellij/common/utils/DateHelperTest.java (hunk 1 above)
Comment (lstocchi) [reply src/test/java/com/redhat/devtools/intellij/common/utils/DateHelperTest.java:86]: Suggestion for the name? What do you mean by negative test? That is the negative test, if you pass a start time that is older than the end time you have a negative result

Comment (jeffmaury) [reply src/test/java/com/redhat/devtools/intellij/common/utils/DateHelperTest.java:86]: so this test should be renamed as `shouldOnlyPrint0SecondsIfStartIsEqualToEnd` and add a new test `shouldOnlyPrint0SecondsIfStartIsOlderThanEnd` when start = end + 1(s|m|h)

Comment (jeffmaury) [inline src/test/java/com/redhat/devtools/intellij/common/utils/DateHelperTest.java:86]: Sorry I misread the test

**Review by jeffmaury — APPROVED** [review]
